docs/reference/configuration.md

```diff
@@ -114,9 +114,13 @@ This reference uses `.` to denote the nesting of values.
 * `blockstore.gs.credentials_file` `(string : )` - If specified will be used as a file path of the JSON file that contains your Google service account key
 * `blockstore.gs.credentials_json` `(string : )` - If specified will be used as JSON string that contains your Google service account key (when credentials_file is not set)
 * `blockstore.gs.pre_signed_expiry` `(time duration : "15m")` - Expiry of pre-signed URL.
+* `blockstore.gs.disable_pre_signed` `(bool : false)` - Disable use of pre-signed URL.
+* `blockstore.gs.disable_pre_signed_ui` `(bool : false)` - Disable use of pre-signed URL in the UI.
```

Comment:
Relevant to all block store that holds this configuration - `diable_pre_signed` needs to `true` to enable this one.